src/SOS/Strike/hostcoreclr.cpp

```diff
@@ -807,17 +820,21 @@ void InitializeSymbolStoreFromSymPath()
 {
     if (g_SOSNetCoreCallbacks.InitializeSymbolStoreDelegate != nullptr)
```

Comment:
2 things:
- I see the other `InitializeSymbolStore` holds a lot of this logic. Can they share it or one call the other so ?
- Same nit about named arguments for default values. 

Also, for the caller (InitializeSymbolStore), it looks like g_symbolStoreInitialized only gets set in windows. Is this expected behavior?

Reply:
The first InitializeSymbolStore with all the arguments really only has the call to the managed function (InitializeSymbolStoreDelegate) in common.  The second InitializeSymbolStore and InitializeSymbolStoreFromSymPath are for Windows only and to set the SymbolStore from the .sympath. I really don't see a lot of common code really. 

I didn't think C++ allowed named arguments.

g_symbolStoreInitialized is set in the first InitializeSymbolStore for both windows/xplat.